server/datastore/mysql/hosts.go

```diff
@@ -706,20 +698,24 @@ func (d *Datastore) EnrollHost(ctx context.Context, osqueryHostID, nodeKey strin
 				return ctxerr.Wrap(ctx, err, "update host")
 			}
 		}
-
+		_, err = tx.ExecContext(ctx, `
+			INSERT INTO host_seen_times (host_id, seen_time) VALUES (?,?)
+			ON DUPLICATE KEY UPDATE seen_time = VALUES(seen_time)`,
```

Comment:
So we were not updating seen_time at host re-enrollment?

Reply:
Yes, looks like it, though that wouldn't explain the missing `host_seen_times` entries... (they should have been created on the first enroll or in the migration...).